client/asset/btc/btc.go

```diff
@@ -1075,6 +1107,37 @@ func (btc *baseWallet) shutdown() {
 	btc.findRedemptionMtx.Unlock()
 }
 
+// Reconfigure attempts to reconfigure the wallet.
+func (btc *baseWallet) Reconfigure(ctx context.Context, cfg *asset.WalletConfig, currentAddress string) (restart bool, err error) {
+	// See what the node says.
+	restart, err = btc.node.reconfigure(cfg, currentAddress)
+	if err != nil {
+		return false, err
+	}
+
+	parsedCfg := new(RPCWalletConfig)
+	if err = config.Unmapify(cfg.Settings, parsedCfg); err != nil {
+		return false, err
+	}
+	walletCfg := &parsedCfg.WalletConfig
+
+	// Make sure the configuration parameters are valid. If restart is required,
+	// this validates the configuration parameters, preventing an ugly surprise
+	// when the caller attempts to open the wallet. If no restart is required,
+	// we'll swap out the configuration parameters right away.
+	newCfg, err := readBaseWalletConfig(walletCfg)
+	if err != nil {
+		return false, err
+	}
+
+	if !restart {
+		// No restart required. Just update the configuration.
+		btc.cfgV.Store(newCfg)
```

Comment:
So we're on the same page, `restart` means this `baseWallet` is going to the trash. (right?) Which means we don't *have* to store the new `baseWalletConfig`.
I guess what I'm getting at is: it could be always:

```go
btc.cfgV.Store(newCfg) // probably trash if restart/reinit required
return restart, nil
```

My thinking is that if we've returned a nil error, and the caller is in charge of restarting... in the meantime... should we apply the new config items assuming the wallet is still live or the caller might not want to restart fully but still take the new settings they requested?